deploy/operator/capi/deploy_capi_cluster.sh

```
@@ -98,7 +98,7 @@ mirror:
   - name: ${RELEASE_IMAGE_HCP_OVERRIDE}
   - name: ${CAPI_IMAGE}
 EOM
-    oc-mirror --config isc.yaml --authfile "${PULL_SECRET_FILE}" --workspace file://$PWD/mirror docker://"${OCP_MIRROR_REGISTRY}" --v2
+    run_mirror_command_with_retry oc-mirror --config isc.yaml --authfile "${PULL_SECRET_FILE}" --workspace file://$PWD/mirror docker://"${OCP_MIRROR_REGISTRY}" --v2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)